ci-operator/step-registry/openshift/image-based/install/openshift-image-based-install-commands.sh

```diff
@@ -51,11 +51,16 @@ export BACKUP_SECRET=\$(<${remote_workdir}/.backup_secret.json)
 
 sudo dnf -y install runc crun gcc-c++ zip
 
+mkdir tmp
+podman run -v ./tmp:/tmp:Z --user root:root --rm --entrypoint='["/bin/sh","-c"]' ${INSTALLER_PULL_REF} "cp /bin/openshift-install /tmp/openshift-install"
+sudo mv ./tmp/openshift-install /usr/bin/openshift-install
+rm -rf tmp
```

Comment:
Maybe we can do something like:
```bash
podman save ${INSTALLER_PULL_REF} | tar -xOf - /bin/openshift-install > /usr/bin/openshift-install
```

instead ?

Reply:
`podman save` outputs the oci-archive layers, but perhaps you were going for a `podman image mount`?

I'm checking if we can go with something like this:

```shell
podman unshare
mnt=podman image mount ${INSTALLER_PULL_REF}cp mnt/bin/openshift-install /usr/bin/openshift-install
```

Reply:
Yeah I wanted to do image mount but it's not supported on rootless containers, but now I remember you're probably running as root here anyway?

Reply:
Could just do `cp $(podman image mount ${INSTALLER_PULL_REF})/bin/openshift-install /usr/bin/openshift-install` and leave the mount

Reply:
> pretty sure you don't want to delete it anyway because sometimes mount gives you a directory that already exists, because of the way podman storage works

No I don't, just tried it :D :D 

Let me try with the simple copy command you came up with :pray:

Reply:
The following didn't succeed (although of course it succeeds on my local workstation :) ):

```shell
podman pull ${INSTALLER_PULL_REF}
podman unshare
cp $(podman image mount ${INSTALLER_PULL_REF})/bin/openshift-install /usr/bin/openshift-install
```

so I've reverted it to the previous solution, in order to get our IBI CI back up to green again. I'll try to improve it in a subsequent PR, WDYT?